PyRDF/backend/Dist.py

```diff
@@ -685,9 +685,95 @@ def ProcessAndMerge(self, mapper, reducer):
         pass
 
     @abstractmethod
-    def distribute_files(self, includes_list):
+    def _distribute_unique_paths(self, paths):
```

Comment:
What does this method do? If it is intended to be implemented by the backends, shouldn't it be not flagged as internal?

Reply:
Here I'm a bit confused about the guidelines for inheritance. It is true that this method is not used directly by the `Dist` class, but at the same time it's "private" because it shouldn't be part of the user-facing API. Maybe it doesn't make a difference and I just misinterpreted the guidelines

Reply:
Ok I guess it's ok because it's meant to be invoked by another method in `Dist` (`distribute_files`) that the subclasses will inherit.

Reply:
Still languages like Java don't allow private + abstract. Python is ok because there is no real private.

Reply:
Ok so in light of these doubts I think it's better to remove the leading underscore